Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=root
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+VOLUME /var/lib/mysql
+
+EXPOSE 3306
```

Comment:
![image](https://github.com/user-attachments/assets/deee0b09-0037-4d5e-98da-2257e4aa9b35)
Add Eof(End-of-file), need to add a blank line at the end